contracts/WitnetRequestBoard.sol

```diff
@@ -74,7 +64,7 @@ contract WitnetRequestBoard is WitnetRequestBoardInterface {
 
     /// @notice Initilizes a centralized Witnet Request Board with an authorized committee.
     /// @param _committee list of authorized addresses.
-    constructor(address[] memory _committee) public {
+    constructor(address[] memory _committee) {
         owner = msg.sender;
         for (uint256 i; i < _committee.length; i++) {
```

Comment:
better be explicit on for clauses initialization: `for (uint256 i = 0; i < _committee.lenght; i++) ...`.